### PR TITLE
Add in Concourse PR pipeline

### PR DESCRIPTION
**Pull request by cybojenix**
Based on https://github.com/cybojenix/govwifi-concourse-pr-template/tree/master/govwifi-pr

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -5,14 +5,27 @@ ifdef DEPLOYMENT
 endif
 
 DOCKER_COMPOSE = docker-compose -f docker-compose.yml
-DOCKER_BUILD_CMD = docker-compose build $(BUNDLE_FLAGS)
 
 ifndef JENKINS_URL
-	DOCKER_COMPOSE += -f docker-compose.development.yml
+  ifndef ON_CONCOURSE
+    DOCKER_COMPOSE += -f docker-compose.development.yml
+  endif
```
Comment (emileswarts) [inline Makefile]: Should these be tabs?

File: ci/pipelines/pr.yml [hunk 2]
```diff
@@ -0,0 +1,101 @@
+resource_types:
+- name: pull-request
+  type: registry-image
+  source:
+    repository: teliaoss/github-pr-resource
+
+resources:
+  - name: pull-request
+    type: pull-request
+    check_every: 1m
```
Comment (emileswarts) [inline ci/pipelines/pr.yml]: We won't run out of github requests if we do this every minute will we?

Comment (paroxp) [reply ci/pipelines/pr.yml]: 1m is usually the default anyway, and if it's a public repo it should be OK.

File: Makefile (hunk 1 above)
Comment (cybojenix) [reply Makefile]: only needs to be tabs when inside rules.

Generally for nesting with definitions, you can use spaces.

Can change this to tabs if you prefer

**Pull request merged**